contracts/programs/contract-reader-interface/src/lib.rs

```diff
@@ -125,6 +141,36 @@ pub struct InitializeMultiReadOnce<'info> {
     pub system_program: Program<'info, System>,
 }
 
+#[derive(Accounts)]
+pub struct InitializeMultiReadWithParamsOnce<'info> {
+    #[account(mut)]
+    pub signer: Signer<'info>,
+
+    #[account(
+        init_if_needed,
+        payer = signer,
+        space = size_of::<MultiRead3>() + 8,
```

Comment:
Suggestion: instead of doing this `size_of::`, the Anchor way of doing this is to:
- Add `#[derive(InitSpace)]` to the struct
- Use `MultRead3::INIT_SPACE` here
See [the docs](https://www.anchor-lang.com/docs/references/space#the-initspace-macro). That way it will also handle some other types, such as vectors (where you can specify a `max_len` to it for example).